src/remote.sh

```diff
@@ -84,10 +92,17 @@ function cp_host2remote()
   dst=${dst:-"$REMOTE_KW_DEPLOY"}
   remote=${remote:-"localhost"}
   port=${port:-"22"}
-  user=${user:-"root"}
+  user=${user:-$(whoami)}
   flag=${flag:-"HIGHLIGHT_CMD"}
 
-  cmd_manager "$flag" "rsync -e 'ssh -p $port' -La $src $user@$remote:$dst"
+  if [[ -v configurations['ssh_configfile'] ]]; then
+    cmd_manager "$flag" "rsync -e 'ssh -F ${configurations['ssh_configfile']}' -La $src ${configurations['hostname']}:$dst --rsync-path='sudo rsync'"
+  else
+    cmd_manager "$flag" "rsync -e 'ssh -p $port' -La $src $user@$remote:$dst --rsync-path='sudo rsync'"
+  fi
+
+  cmd_remotely "sudo chown -R root:root $dst" "$flag" "$remote" "$port" "$user"
```

Comment:
\* Then we would have something as `sudo sudo` here?

Reply:
oops, you are right. I though I had fixed that, thank you! I will update it

Reply:
done